templates/admin/usergroup/list.html.twig

```diff
@@ -0,0 +1,152 @@
+{#
```

Comment:
there's a strange file above templates/admin/usergroup/list.html.twig that seems to be empty maybe?